projects/cm_mcu/LocalTasks.c

```diff
@@ -656,3 +656,230 @@ void init_registers_ff()
   apollo_i2c_ctl_reg_w(3, 0x21, 1, 0x03, 1, 0x01); //  00000001 [P17..P10]
 }
 #endif // REV2
+
+#ifdef REV2
+int init_load_clk(int clk_n)
+{
+
+  if (getPowerControlState() != POWER_ON) {
+	  log_debug(LOG_SERVICE, "LGA80D_init() needs more time to turn on power supplies \r\n");
+	  vTaskDelay(pdMS_TO_TICKS(3000)); // wait 3s
+  }
+  clk_n = clk_n -1;
+  int PreambleList_row[5] = {6,0,3,0,0};
+  int RegisterList_row[5] = {378,0,587,0,0};
+  int PostambleList_row[5] = {3,0,5,0,0};
+  char *clk_ids[5] = {"r0a","r0b","r1a","r1b","r1c"};
+  uint8_t i2c_mux_masks[5] = {0x01,0x02,0x04,0x08,0x10};
+  uint8_t i2c_addrs[5] = {0x77,0x6b,0x6b,0x6b,0x6b};
+
+  //apollo_i2c_ctl_w(2, 0x70, 1, 0xc0);
+  apollo_i2c_ctl_w(2, 0x70, 1, i2c_mux_masks[clk_n]);
```

Comment:
what happens if this call fails? also please remove magic numbers and delete commented-out line